pkg/proxy/ipvs/proxier.go

```diff
@@ -1867,6 +1859,11 @@ func (proxier *Proxier) createAndLinkeKubeChain() {
 			klog.Errorf("Failed to ensure that %s chain %s exists: %v", ch.table, ch.chain, err)
 			return
 		}
+		// KUBE-MARK-MASQ and KUBE-MARK-DROP is created in kubelet
+		// kube-proxy only ensure it exist but not do anything to those chains
+		if ch.chain == KubeMarkDropChain || ch.chain == KubeMarkMasqChain {
```

Comment:
Instead of checking specifically for KubeMarkDropChain and KubeMarkMasqChain here and skipping, I would prefer a separate list of chains containing `KubeMarkDropChain` and `KubeMarkMasqChain` only (similar to iptablesChains) where we only run `proxier.iptables.EnsureChain`.

Reply:
Sure, that might be better. Addressed. : )